src/Message/Communicate.cpp

```diff
@@ -37,10 +37,6 @@ Communicate* OHMMS::Controller = new Communicate;
 //default constructor: ready for a serial execution
 Communicate::Communicate() : myMPI(MPI_COMM_NULL), d_mycontext(0), d_ncontexts(1), d_groupid(0), d_ngroups(1) {}
 
-#ifdef HAVE_MPI
-Communicate::Communicate(const mpi3::environment& env) { initialize(env); }
```

Comment:
Should delete the corresponding declaration in Communicate.h

Reply:
yes, please. That is correct.